srcpkgs/nextcloud-client/template

```diff
@@ -41,6 +42,16 @@ fi
 
 subpackages="$(vopt_if dolphin 'nextcloud-client-dolphin') nextcloud-client-devel"
 
+if [ "XBPS_CHECK_PKGS" ]; then
```

Comment:
Missing dollar.

Reply:
thanks ! related to end of week I guess ...